src/schema/v2/sellable.ts

```diff
@@ -21,6 +21,12 @@ export const Sellable = new GraphQLInterfaceType({
       description: "Whether a user can make an offer on the work",
       resolve: ({ is_offerable }) => is_offerable,
     },
+    isOfferableFromInquiry: {
+      type: GraphQLBoolean,
+      description:
+        "Whether a user can make an offer on the work through inquiry",
+      resolve: ({ is_offerable_from_inquiry }) => is_offerable_from_inquiry,
```

Comment:
I see this is following the pattern above but I wonder if we want the field from Gravity `offerable_from_inquiry` instead of `is_offerable_from_inquiry`, like the [artwork type](https://github.com/artsy/metaphysics/pull/2952/files#diff-54b84e0c164089fc202e3439d8ded89ab4925442fb613f4e36a9c22acdb66cb8R397).

And actually I am curious if the resolver in an interface is used. 🤔

Reply:
🐈 The test passes so it should work just fine but does `... => is_offerable_from_inquiry` need to be changed to  `resolve: ... => offerable_from_inquiry`?